packages/experimental/Icon/src/Icon.types.ts

```diff
@@ -0,0 +1,33 @@
+import { SvgProps } from 'react-native-svg';
+import { AccessibilityProps } from 'react-native';
+export const iconName = 'Icon';
+
+export interface IFontIconProps
+{
+  fontFamily?: string;
+  fontSrcFile?: string;
+  codepoint: number;
+  fontSize?: number;
+}
+
+export interface ISvgIconProps
+{
+  uri?: string;
+  src?: React.FC<SvgProps>;
+  viewBox?: string;
+}
+
+export interface IRasterImageIconProps
+{
+  src: object;
+}
+
+export interface IIconProps extends AccessibilityProps
+{
+  fontSource?: IFontIconProps;
+  svgSource?: ISvgIconProps;
+  rasterImageSource?: IRasterImageIconProps;
+  width?: number;
```

Comment:
Can we add a more standard sizing pattern, maybe something like what is available for PersonaCoin, or some sort of set of 'small' | 'medium' | 'large' etc? Possibly move the PersonaCoin mapping to a new shared location. Specific width/height values could be set via style but likely shouldn't be the primary way to set this.